_data/conferences.yml

```diff
@@ -0,0 +1,1151 @@
+##########################
+#
+# Use this file to add a new conference to the list of past/upcoming Cococa conferences
+#
+# All conferences must provide a name.
+#
+# If the "link" is left out, then the conference name will not link to anywhere.
+#
+# If the "location" is left out, it will display as "TBA".
+#
+# If the "start" date is left out, the conference date will display as "TBA".
+#
+# If the "end" date is left out, it will be assumed to be the same as the start date (ie, either "TBA" or a one-day conference)
+#
+# If "cocoa" is left out, it will be assumed to be "false" (ie, it'll be listed as a "mobile development in general" conference)
+#
+# If both the CFP link and deadline are included, the date will be formatted as the link text. If the date has passed, it will also be struck-through.
+# If only the CFP link is included, the link text will be "Deadline not specified"
+# If only the CFP deadline is included, the date will be formatted. There will not be a link.
+# If both the CFP deadline and link are omitted, the text will say "See website for details". There will not be a link.
+#
+# Special characters (like leading @ symbols) may need to be escaped. You can do so using a backslash. Eg: \@Swift
+#
+##########################
+
+- name: Example Con 2019
```

Comment:
Is this actually evaluated? or skipped because its missing some information?

Reply:
This was a test thing to make sure my changes would work. I just took it out in a new commit.